presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableScanNode.java

```diff
@@ -50,13 +50,16 @@
 
     private final TupleDomain<ColumnHandle> enforcedConstraint;
 
+    private final boolean temporaryTable;
```

Comment:
nit: This variable name sounds a bit off to me. But I don't have a good suggestion ('scanFromTemporaryTable`? )

Any thoughts, @rschlussel , @tdcmeehan ? If I was the only one feel off then let's just keep this name :)

Reply:
I agree this looks ugly. Hopefully we will get rid of this once we refactor scheduling, so split sources are opened during scheduling and not during planning.